frontend/src/components/pages/connect/helper.tsx

```diff
@@ -594,11 +593,20 @@ export const ConfirmModal = observer(<T,>(props: ConfirmModalProps<T>) => {
             <AlertDialogHeader>Confirm</AlertDialogHeader>
             <AlertDialogBody>
                 {content}
-                {err && <Box mt={4}><Alert
-                    type="error"
-                    message={err.title}
-                    description={err.content}
-                /></Box>}
+                {err && <Box mt={4}>
+                     <Alert
+                         status="error"
+                         variant="left-accent"
+                     >
+                         <AlertIcon />
+                         <AlertDescription>
+                             <Box>
+                                 <Text as="h3">{err.title}</Text>
+                                 <Text as="p">{err.content}</Text>
```

Comment:
I don't think you need the `as="p"`, if I remember correctly, by default `Text` renders a paragraph

Reply:
Yup. `<Text>` defaults to a `<p>` tag.